src/Frontend/Modules/Faq/Widgets/AskOwnQuestion.php

```diff
@@ -55,19 +55,19 @@ public function execute()
     }
 
     /**
-     * Load the form
+     * Load the form.
      */
     private function loadForm()
     {
         // create form
         $this->frm = new FrontendForm('own_question', '#' . FL::getAction('OwnQuestion'));
-        $this->frm->addText('name')->setAttributes(array('required' => null));
-        $this->frm->addText('email')->setAttributes(array('required' => null, 'type' => 'email'));
-        $this->frm->addTextarea('message')->setAttributes(array('required' => null));
+        $this->frm->addText('name')->setAttributes(['required' => null]);
```

Comment:
I've remove the previous commit since it didn't actually do anything besides the array syntax